plugins/modules/helm_repository.py

```diff
@@ -275,14 +292,15 @@ def main():
         helm_cmd = delete_repository(helm_cmd, repo_name)
         changed = True
     elif repo_state == "present":
-        if repository_status is None:
+        if repository_status is None or force_update:
```

Comment:
My concern is that even if the specs are exactly the same, we will still mark it as changed, on the other side, in helm it works the same, for example:
```
➜ helm repo add stable https://kubernetes.github.io/ingress-nginx
"stable" already exists with the same configuration, skipping
➜ helm repo add stable https://kubernetes.github.io/ingress-nginx --force-update
"stable" has been added to your repositories
```

Reply:
Alright, thanks.